packages/twenty-front/src/modules/apollo/hooks/useApolloFactory.ts

```diff
@@ -30,7 +30,13 @@ export const useApolloFactory = (options: Partial<Options<any>> = {}) => {
   const apolloClient = useMemo(() => {
     apolloRef.current = new ApolloFactory({
       uri: `${REACT_APP_SERVER_BASE_URL}/graphql`,
-      cache: new InMemoryCache(),
+      cache: new InMemoryCache({
+        typePolicies: {
+          RemoteTable: {
+            keyFields: ['name'],
```

Comment:
TODO: Check if we have an issue when we migrate between the show page of two databases that have a table with the same name. If so we could add connectionId to the cache id to make it unique